File: src/main/java/com/wisenut/infrastructures/repository/kakao/KaKaoMapInfoRepository.java
package com.wisenut.infrastructures.repository.kakao;


import com.wisenut.domain.model.kakaomap.KaKaoMapInfo;
import org.springframework.data.jpa.repository.JpaRepository;

public interface KaKaoMapInfoRepository extends JpaRepository<KaKaoMapInfo,Long>, KaKaoMapInfoRepositoryCustom {

}

File: src/main/java/com/wisenut/domain/model/kakaomap/KaKaoMapInfoRepository.kt
package com.wisenut.domain.model.kakaomap

import com.wisenut.infrastructures.repository.kakao.KaKaoMapInfoRepositoryCustom
import org.springframework.data.jpa.repository.JpaRepository

interface KaKaoMapInfoRepository : JpaRepository<KaKaoMapInfo?, Long?>, KaKaoMapInfoRepositoryCustom